src/components/NavigationStepper/index.tsx

```diff
@@ -63,30 +63,30 @@ const StyledStep = styled('li', {
   ${({ condensed, isCompleted, theme }) =>
     isCompleted
       ? css`
-          color: ${theme.colorsDeprecated.success};
+          color: ${theme.colors.success.text};
           ${StyledStepNumber} {
-            background-color: ${theme.colorsDeprecated.success};
-            color: ${theme.colorsDeprecated.white};
+            background-color: ${theme.colors.success.backgroundStrong};
+            color: ${theme.colors.success.textStrong};
           }
         `
       : css`
-          color: ${theme.colorsDeprecated.gray550};
+          color: ${theme.colors.neutral.textWeak};
           ${StyledStepNumber} {
-            border: 3px solid ${theme.colorsDeprecated.gray550};
+            border: 3px solid ${theme.colors.neutral.borderWeak};
           }
 
           &[aria-current='true'] {
-            color: ${theme.colorsDeprecated.primary};
+            color: ${theme.colors.primary.text};
```

Comment:
```suggestion
            color: ${theme.colors.primary.textWeak};
```

as background is in a weak context (white), from now they are both same color but later it will be useful :)